lib/registries/addon/edit-revision/revision-manager.ts

```diff
@@ -72,22 +72,22 @@ export default class RevisionManager {
 
     @computed('revision.reviewState')
     get isEditable() {
-        return this.revision.reviewState === RevisionReviewStates.RevisionInProgress;
+        return this.revision && this.revision.reviewState === RevisionReviewStates.RevisionInProgress;
```

Comment:
Note to self: are these checks still needed? either way, can rewrite this as `this.revision?.reviewState` I think?